src/assessment/migrations/0007_affectivecheckresponse.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 2.2.20 on 2021-04-23 17:14
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('roster', '0031_clusiveuser_education_levels'),
+        ('library', '0032_auto_20210416_1809'),
+        ('assessment', '0006_auto_20210415_2101'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='AffectiveCheckResponse',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('created', models.DateTimeField(auto_now_add=True)),
+                ('updated', models.DateTimeField(auto_now=True)),
+                ('annoyed_option_response', models.BooleanField(null=True)),
```

Comment:
I'm wondering whether these should be null=False.
When the form is submitted, the words are either selected or not.

Reply:
Agree; I've set `null=False` and `default=False`.